docs/sources/panels-visualizations/visualizations/canvas/index.md

```diff
@@ -85,6 +85,18 @@ You can edit your canvas inline while in the context of dashboard mode.
 
 {{< video-embed src="/static/img/docs/canvas-panel/canvas-inline-editor-9-2-0.mp4" max-width="750px" caption="Inline editor demo" >}}
 
+### Pan and zoom
+
+You can enable panning and zooming in canvas. This allows you to both create and navigate more complex designs.
+
+{{< docs/public-preview product="Canvas pan and zoom" featureFlag="canvasPanelPanZoom" >}}
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10-3-canvas-pan-zoom-setting.png" max-width="300px" caption="Canvas pan zoom control" >}}
```

Comment:
Let's make these alt text rather than captions since alt text is required and captions aren't
```suggestion
{{< figure src="/media/docs/grafana/screenshot-grafana-10-3-canvas-pan-zoom-setting.png" max-width="300px" alt="Canvas pan zoom control" >}}
```